src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UnitTest/UnitTestsControl.cs

```diff
@@ -230,14 +343,14 @@ private Color GetTestResultColor(TestResult testResult)
 				case TestResult.Ignored:
 					return Colors.Orange;
 
-				case TestResult.Sucesss:
+				case TestResult.Passed:
 					return Colors.LightGreen;
 			}
 		}
 
 		private async Task RunTests(CancellationToken ct, string[] filters)
 		{
-			(int run, int ignored, int succeeded, int failed) counters = (0, 0, 0, 0);
+			_currentRun = new TestRun();
```

Comment:
We will probably have some conflicts here 😀

Reply:
Yes, I don't mind :) I'll redo mine, yours is more important to merge.